apps/src/applab/applab.js

```diff
@@ -763,6 +769,10 @@ Applab.onMouseMoveDebugResizeBar = function (event) {
                        Math.min(MAX_DEBUG_AREA_HEIGHT,
                                 (window.innerHeight - event.pageY) - offset));
 
+  if (debugAreaController.isShut()) {
+    debugAreaController.snapOpen();
```

Comment:
it looks from the anigif that the debug bar snaps open when you click the resize bar. does it gradually open if you drag (starting from closed)?